docs/ru/1581.md

```diff
@@ -0,0 +1,30 @@
+# Ретранслятор на базе клевера
+
+[CopterHack-2022](copterhack2022.md), команда **команда Джедаи 1581**.
+
+## Информация о команде
+
+Состав команды:
+* Никульшина Юлия, капитан, СМИ, инженер t.me/JJNikK.
```

Comment:
This would not show as a link, you should add https:// or make a link like this:

```suggestion
* Никульшина Юлия, капитан, СМИ, инженер [@JJNikK](https://t.me/JJNikK).
```